src/main/VirtuDoc - Records Pages/Back-End/demo/src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.example.demo;
```

Comment:
I might be mis-understanding what's going on here, but why do we need a second Spring application? Why not use the existing Spring stuff located at the project root? The Spring application at the project root already has database capabilities and is already configured to talk to the Postgres instance that runs in the dev stack.